hoptimator-cli/run.sh

```diff
@@ -5,5 +5,6 @@ java \
   --add-opens java.base/java.util=ALL-UNNAMED \
   --add-opens java.base/java.time=ALL-UNNAMED \
   -classpath "/opt/plugins/*/lib/*:./hoptimator-cli-all.jar" \
+  -Dorg.slf4j.simpleLogger.defaultLogLevel=error \
```

Comment:
should `error` be in UPPER-CASE `ERROR`?

How to override to other levels for debugging?

Reply:
Lowercase is correct. For specific classes, you can use `org.slf4j.simpleLogger.log.a.b.c=debug` etc. See https://www.slf4j.org/api/org/slf4j/simple/SimpleLogger.html.

Also see #36. @HoganEdwardChu proposed switching to log4j. I don't have a strong preference, but I don't think log4j is worth the complexity.